app/hotels/src/filter/Filters.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+
+import * as React from 'react';
+
+type Props = {|
+  +children: React.Node,
+|};
+
+export default function Filters(props: Props) {
+  const activeFilters = [];
+  const inActiveFilters = [];
```

Comment:
If I understand correctly, this should read _inactive_ and not _in active_, right? In that case, I would replace `inActive` by `inactive`.

Reply:
you are of course right, i will change it to lowercase `a`